diff --git a/node_modules/eslint-mdx/lib/worker.js b/node_modules/eslint-mdx/lib/worker.js
index 7dd21a5..adaabcc 100644
--- a/node_modules/eslint-mdx/lib/worker.js
+++ b/node_modules/eslint-mdx/lib/worker.js
@@ -8,6 +8,7 @@ const path_1 = tslib_1.__importDefault(require("path"));
 const cosmiconfig_1 = require("cosmiconfig");
 const synckit_1 = require("synckit");
 const helpers_1 = require("./helpers");
+const { pathToFileURL } = require("url");
 let acorn;
 let tokTypes;
 let jsxTokTypes;
@@ -129,7 +130,7 @@ exports.getRemarkProcessor = getRemarkProcessor;
         })(acorn.Parser).acornJsx.tokTypes;
     }
     if (!TokenTranslator) {
-        TokenTranslator = (yield (0, helpers_1.loadEsmModule)(path_1.default.resolve(require.resolve('espree/package.json'), '../lib/token-translator.js'))).default;
+        TokenTranslator = (yield (0, helpers_1.loadEsmModule)(pathToFileURL(path_1.default.resolve(require.resolve('espree/package.json'), '../lib/token-translator.js')))).default;
     }
     const tokenTranslator = new TokenTranslator(Object.assign(Object.assign({}, tokTypes), jsxTokTypes), fileOptions.value);
     const root = processor.parse(fileOptions);
